src/pkg/auth/auth.go

```diff
@@ -160,30 +168,57 @@ func StartAuthCodeFlow(ctx context.Context, mcpFlow LoginFlow) (AuthCodeFlow, er
 	if mcpFlow {
```

Comment:
Not necessary to change anything in this PR, but I'm starting to think this should just be the default flow. Try to open a browser, if it fails, print a url. I don't see the value in waiting for a keypress anymore.